templates/aruba_os_show_aaa_authentication_all.template

```diff
@@ -0,0 +1,12 @@
+Value METHOD (Web|802.1x|Management|tacacs-accounting)
```

Comment:
I think we should be safe to use `(\S+)` here, and that would allow for additional options or variations in formats (capitalization, hyphens, abbreviations).